frontend-mini-challenge/components/Text.tsx

```diff
@@ -1,14 +1,14 @@
 import { Text } from "@mantine/core";
 
-export function Bold({ children }) {
+export function Bold({ children }: { children: React.ReactNode }) {
   return (
     <Text fw={700} component="span">
       {children}
     </Text>
   );
 }
 
-export function Underlined({ children }) {
+export function Underlined({ children }: { children: React.ReactNode }) {
```

Comment:
Look at how these functions are used. Are you sure they take in any ReactNode or are they called with one specific type of node